free-2.md

```diff
@@ -58,12 +105,25 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+A lot of the time would be spent on figuring out good ways to interact with the
+user. Keeping track of events is pretty easy and notifications can be anywhere
+from easy to pretty difficult. However, the user interaction can also be pretty
+incremental: adding support for manual input would be one level, gmail input
+would be harder, and other methods of input and output could get trickier.
```

Comment:
For  MVP, you probably wouldn't even need notifications. Just require the user to ask "Hello there, task tracker I've been ignoring because I don't want to face reality. What have I forgotten?"